.github/workflows/build.yml

```diff
@@ -96,7 +96,7 @@ jobs:
           dev/ci/util_free_space.sh
 
       - name: Build with Gradle
-        run: ./gradlew build -PskipITs -PjdkVersion=${{ matrix.java-version }} -x :clients:client-python:build
+        run: ./gradlew build -PskipITs -PjdkVersion=${{ matrix.java-version }} -PskipDockerTests=false -x :clients:client-python:build
```

Comment:
Why do we enable docker tests for build task?

Reply:
build task needs to run UTs, some UTs include docker tests(such as Doris catalog)